sql2.txt

```diff
@@ -2,44 +2,45 @@
 to join the OrderDetails table with the product table.  For TotalCount, you will Use
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
-
+SELECT p.ProductName,SUM(od.Quantity) as TotalCount FROM Products p JOIN OrderDetails od ON p.ProductID = od.ProductID GROUP BY p.ProductName;
 
 2. Retrieve the customer name and OrderDetailsID for all the OrderDetails for each customer,
 ordered by customer name. Paste your SQL statement below. You will have to join the Customers
 table with the Orders table, and the Orders table with the OrderDetails table, 
 because the Orders table is the one that ties the Customer table to the OrderDetails table.
-
+SELECT c.CustomerName,od.OrderDetailID FROM Customers c JOIN OrderDetails od ON c.CustomerID = o.CustomerID JOIN Orders o ON o.OrderID = od.OrderID ORDER BY c.CustomerName;
 
 3. Extend the previous query to join the Products table.  You are now joining the Customers, Orders,
 OrderDetails, and Products tables.  The result of your query should include the
 customer name, the product name, and the total quantity of that product ordered.  Note that a customer
 may have ordered the same product in several different orders.  So the total quantity is
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
-
+SELECT c.CustomerName,p.ProductName,SUM(od.Quantity) AS TotalQuantity FROM Customers c JOIN OrderDetails od ON c.CustomerID = o.CustomerID JOIN Orders o ON o.OrderID = od.OrderID JOIN Products p ON p.ProductID = od.ProductID GROUP BY c.CustomerName,p.ProductName;
 
 
 4. Extend the previous query to give the total price as well as the customer name, product name, 
 and total quantity.  Here you will have
 (SUM(Quantity) * Price) AS TotalPrice.  Paste your SQL statement below.
-
+SELECT c.CustomerName,p.ProductName,SUM(od.Quantity) AS TotalQuantity,SUM(od.Quantity * p.Price) AS TotalPrice FROM Customers c JOIN OrderDetails od ON c.CustomerID = o.CustomerID JOIN Orders o ON o.OrderID = od.OrderID JOIN Products p ON p.ProductID = od.ProductID GROUP BY c.CustomerName,p.ProductName;
 
 5. Add a new Order to the Orders table.  You will have to specify the ID of an existing customer
 to satisfy the foreign key constraint.  You will also have to specify the ID of an existing
 employee.  Paste your SQL statement below.
-
+INSERT INTO Orders (CustomerID,EmployeeID,OrderDate,ShipperID) VALUES (3,5,"7/11/1996",2);
 
 6. Add three OrderDetails.  These should specify the ID of the new Order you just created.  For each,
 you will have to specify the ID of an existing product.  Paste your SQL statement below.
+INSERT INTO OrderDetails (OrderID,ProductID,Quantity) VALUES (10444,3,5),(10444,4,10),(10444,5,1);
```

Comment:
Functionality is correct just need some spaces between  `(OrderID, ProductID, Quantity)`